src/services/game-logic/pieces/Bishop.ts

```diff
@@ -0,0 +1,27 @@
+import { PieceNames } from '../../../enums';
+import { Square } from '../../../models/Square';
+import { Piece } from './Piece';
+
+class Bishop extends Piece {
+    name = PieceNames.BISHOP;
+    getPossibleMoves(): Square[] {
+        const positions: Square[] = [];
+        for (let i = 0; i <= 7; i++) {
+            for (let j = 0; j <= 7; j++) {
```

Comment:
consider renaming iterators into `row` and `col` or something more meaningfull

Reply:
Jak dla mnie i, j są najbardziej czytelne jeśli chodzi o operacje na macierzach- nawyki ze studiów

Reply:
Jak dla mnie w tym wypadku litery mogą zostać.